tests/queries/0_stateless/03276_index_of_assume_sorted.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS test;
+
+CREATE TABLE test(
+    id UInt64,
+    numbers Array(Int64)
+)
+ENGINE = MergeTree()
+ORDER BY id;
+
+INSERT INTO test VALUES(1, [1, 2, 2, 3, 3, 3, 4, 4, 4, 5, 6, 7]);
```

Comment:
The docs make a mention "If the internal array type is Nullable, the ‘indexOf‘ function will be used.". Why is that case not tested here?